x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/all/exceptions/exceptions_table.tsx

```diff
@@ -67,15 +90,28 @@ export const ExceptionListsTable = React.memo<ExceptionListsTableProps>(
       notifications,
       showTrustedApps: false,
     });
-    const [loadingTableInfo, data] = useAllExceptionLists({
-      exceptionLists: exceptions ?? [],
-    });
+    const [loadingTableInfo, exceptionListsWithRuleRefs, exceptionsListsRef] = useAllExceptionLists(
+      {
+        exceptionLists: exceptions ?? [],
+      }
+    );
     const [initLoading, setInitLoading] = useState(true);
     const [lastUpdated, setLastUpdated] = useState(Date.now());
     const [deletingListIds, setDeletingListIds] = useState<string[]>([]);
     const [exportingListIds, setExportingListIds] = useState<string[]>([]);
     const [exportDownload, setExportDownload] = useState<{ name?: string; blob?: Blob }>({});
 
+    const handleDeleteSuccess = useCallback(() => {
+      notifications.toasts.addSuccess({ title: i18n.EXCEPTION_DELETE_SUCCESS });
```

Comment:
nit: Optionally add the exception list name to the toast message

Reply:
Added the `list-id` to the toast!
<img width="346" alt="Screen Shot 2021-01-05 at 12 09 29 AM" src="https://user-images.githubusercontent.com/10927944/103608875-6cd2cc00-4eea-11eb-862c-6926a61aceef.png">

Also updated error toast to include more detailed message:
<img width="861" alt="Screen Shot 2021-01-04 at 11 55 57 PM" src="https://user-images.githubusercontent.com/10927944/103608930-97bd2000-4eea-11eb-99bf-7a574bcb1b20.png">